typescript-test-samples/async-architectures/async-lambda-dynamodb/src/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "target": "es2020",
+    "strict": true,
+    "preserveConstEnums": true,
+    "noEmit": true,
+    "sourceMap": false,
+    "module":"es2015",
+    "moduleResolution":"node",
+    "esModuleInterop": true, 
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,  
```

Comment:
Please add 

```json
"typeRoots": [
      "./node_modules/@types"
  ]
```
to resolve IDE error in `app.ts` trying to import `aws-lambda`

See https://github.com/aws-samples/serverless-test-samples/blob/b817a5153b12748fdc82e0f85b0bb7ac6ea63bbc/typescript-test-samples/typescript-test-intro/list-buckets/tsconfig.json#L13 as example

Reply:
Fixed